.travis.yml

```diff
@@ -60,6 +60,6 @@ install:
   - pip install coveralls
 script:
    - PYTHONPATH=".:tests:$PYTHONPATH" python -Wall -m coverage run --omit='setup.py' --source=. tests/manage.py test core --settings=
-   - if python -c 'import sys; sys.exit(1 - (sys.version_info >= (3, 6)))'; then isort --check-only import_export tests; fi
+   - if python -c 'import sys; sys.exit(1 - (sys.version_info >= (3, 6)))'; then isort --check-only -rc import_export tests; fi
```

Comment:
Can we remove the `rc` from here? Its adding a bit of noise to the PR. If we want this in isort, we should add it in another PR.